### PR TITLE
Task 582 Natural Questions

### PR DESCRIPTION
**Pull request by cosmicishan**
task582_natural_question_answer_generation -> You are given an open-domain question and return an answer based on factual information

### CODE REVIEW

**Comment by swarooprm** [comment]
@colinzhaoust, wondering if you can help review this PR?

**Comment by cosmicishan** [comment]
Also, please suggest to me another task I can create from this dataset.
Thanks in advance!!

File: tasks/task352_coda-19_classification.json [hunk 1]
```diff
@@ -0,0 +1,1700 @@
+{
```
Comment (danyaljj) [inline tasks/task352_coda-19_classification.json]: is this file really part of this PR?

**Comment by danyaljj** [comment]
Looks good! Leaving it here for @swarooprm before merging it.

**Review by swarooprm — APPROVED** [review]
LGTM!

There is a minor comment which you can fix in your next PR.
"The short answer can be one or more entities or it can also be boolean \\*yes\\* or \\*no\\*."-->The short answer can be one or more entities. It can also a be boolean: \\*yes\\* or \\*no\\*.

Re: more task suggestion- See if you can create a jeopardy type task here.

**Pull request merged**